Automattic/woocommerce-payments
# Hide Fraud info banner until first transaction happens

Fixes #8279 

#### Changes proposed in this Pull Request


**Hide Fraud info banner until first transaction happens**

To avoid overwhelming new users, we'll be displaying the informational banner about the fraud tools only after they've processed volume in their store using WooPayments.




#### Testing instructions





* Go [to your client phpMyAdmin](http://localhost:8083/index.php?route=/sql&pos=0&db=wordpress&table=wp_options) and make sure you don't have `wcpay_frt_discover_banner_settings` in the table wp_options. If you have it, delete it.

**No transactions**
* Go to [this line](https://github.com/Automattic/woocommerce-payments/blob/4fa44519a2e5da4b723c01e3ba92f513790a4db2/includes/admin/class-wc-payments-admin.php#L857) and modify it 
`'lifetimeTPV'                   => 0,`
* Go to Payments
* The Fraud tools banner is not appearing

**With transactions**
* Remove the modifications that you added in the last step (assuming your test account already has volume in it, if not, feel free to modify it with any number that is not `0`)
* Go to Payments
* The Fraud tools banner is appearing
 
![Screenshot 2024-04-15 at 12 11 37](https://github.com/Automattic/woocommerce-payments/assets/2612426/2312ebdd-4327-42a2-84b3-9fe1ab9c6b97)

* Click on Dismiss
* The Fraud tools banner is not appearing
* Refresh
* The Fraud tools banner is not appearing

-------------------


**Post merge**
